### DISCUSSION TITLE
Dashboard and EDA functions not found in (regression) module

### DISCUSSION THREAD

**Discussion by ReinvanLennep** [pycaret/pycaret#2176] [Q&A] [answered]
Hi,

I installed the full version (3.0.0) of Pycaret. However, when I wanted to call the `eda` and `dashboard` functions, it gave me the following error: 

> `AttributeError: module 'pycaret.regression' has no attribute 'dashboard'`

While I know that the library contains these functions for both the regression and classification modules, I found that in my venv it seems like they are not present. I included a screenshot for more info.

![image](https://user-images.githubusercontent.com/68655766/154695212-f2fa433a-0656-42cc-95cb-0bd06c96087e.png)

Also, it does not matter how I import the modules:
![image](https://user-images.githubusercontent.com/68655766/154695593-e1331df2-14df-47d2-adb9-a65c6a916eeb.png)

I hope somebody can help me on my journey to create my ML pipeline using Pycaret. Untill now, I have a great experience using it :)

**Comment by ngupta23** [comment, marked as answer]
I think you have the pycaret time series module installed. The official 3.0 release for main pycaret has not happened yet. For now, you should do the following to get the dashboard functionality. Remember to do this in a separate environment from the time series module since they are not compatible for now until the official 3.0 release of pycaret is made.

```
pip install pycaret
```

**Reply by ReinvanLennep** [reply]
Hi @ngupta23,

Thank you for your quick reaction! Because of your help I managed to fix it. 

However, I noticed that every time I created a new environment and installed Pycaret[full], I got version 3.0.0. Maybe I did sth wrong or maybe something goes wrong with the installation of the library. Just wanted to inform you on that.

Nonetheless, thx for the help. Problem solved!

**Reply by ngupta23** [reply]
That should be ok. The idea was (and still is) that we will combine the time series module (pycaret-ts-alpha) with the main pycaret package in 3.0.0 so we versioned the current time series module (pycaret-ts-alpha) as 3.0.0

Sorry for the confusion it might cause.
